docs/linter_settings.rst

```diff
@@ -49,11 +50,7 @@ or the file's directory if it is not contained within a project folder
 
 .. note::
 
-     If the value of ``chdir`` is unspecified (or inaccessible), then:
-
-     - If linting a saved file, the directory is set to that of the linted file
-
-     - If linting an unsaved file, the directory is unchanged
+     The above example is the default.
```

Comment:
Imo it should still be mentioned what happens if the resulting `working_dir` ends up being empty, i.e. it won't be changed. With the default setting, this is only relevant for unsaved views, but if the setting is modified and `$file_path` is removed, for example, this could occur more frequently.

Reply:
Here, I dunno. If you set `working_dir` to a falsy value, the default kicks in. Any truthy value passes the first if check. If `working_dir` is set to something which is not a directory we will err and just not use this as `cwd`. We pass `None`.

Generally, unsaved files are not a problem. Only editing an unsaved file in a window without any open folder.